pages/Getting Started Staff.md

```diff
@@ -44,11 +43,17 @@ For example, if user bobby's message count is 20 and you call `/resetlevel bobby
 be 15. If you instead called `/resetlevel bobby 27`, bobby's message count would be set to 27. Lastly, if you called
 `/resetlevel bobby`, bobby's message count would be reset to 0.
 
+![A gif of the resetlevel command being called on Josh, subtracting 10 from his message count][1]
```

Comment:
```suggestion
![A gif of the resetlevel command being called on Josh, subtracting 10 from his message count.][1]
```

Reply:
Nvm; I've realized most of them don't have periods. I thought you just missed a few. Ignore the ones I resolve, please